### PR TITLE
Adding color page

### PR DESCRIPTION
**Pull request by auareyou**

### CODE REVIEW

**Comment by broccolini** [comment]
Hi, you can hack the table width with using non breaking spaces. Edit this comment to see and example. I think you only need to do it one row. Not the nicest solution but works 🤷‍♀️ 

| title | longer title                                                 |
| ------------------------------------------------------------ | ------------------------------------------------------------ |
| `col-text-orange-blah` &nbsp; &nbsp; &nbsp; &nbsp;                                     | flkds fjdkslkkd djksldsl fjdkl;dsla fjkds lskdjfdlsk fdjklsdflkj fierawoeirelkjds kf;alkj f jd;alkfdlksd djdkowwek fjdkls |
| `col-text-yellow-blah`                              | flkds fjdkslkkd djksldsl fjdkl;dsla fjkds lskdjfdlsk fdjklsdflkj fierawoeirelkjds kf;alkj f jd;alkfdlksd djdkowwek fjdkls gjdk;adlkfjsdjsl jfdk;slfkdjs |

**Review by yaili — APPROVED** [review]
:shipit:

File: content/foundations/color.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: Color
+---
+
+Color plays a critical role in an interface’s usability. It helps the user know which actions to take, and understand hierarchy and relationships between elements.
+Primer’s color system is composed of different color modes, with their own color scale. Colors are named based on the function they perform in the interface.
+
+<img width="960" alt="modes" src="https://user-images.githubusercontent.com/6951037/127494388-77f3132f-67fc-49de-89d6-28730f8d3d27.png">
```
Comment (colebemis) [inline content/foundations/color.mdx:8]:
```suggestion
<img width="960" alt="modes" src="https://user-images.githubusercontent.com/6951037/127494388-77f3132f-67fc-49de-89d6-28730f8d3d27.png" />
```

File: content/foundations/color.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Color plays a critical role in an interface’s usability. It helps the user know which actions to take, and understand hierarchy and relationships between elements.
+Primer’s color system is composed of different color modes, with their own color scale. Colors are named based on the function they perform in the interface.
+
+<img width="960" alt="modes" src="https://user-images.githubusercontent.com/6951037/127494388-77f3132f-67fc-49de-89d6-28730f8d3d27.png" />
+
+## Color modes
+GitHub's UI can be rendered in a variety of different color modes. Every element on GitHub’s interface maps to a color in the scale for each color mode.
+
+<img width="960" alt="functional-demo" src="https://user-images.githubusercontent.com/6951037/127494554-07b67dbd-66d3-4d36-8350-5ec837d6d1a5.png">
```
Comment (colebemis) [inline content/foundations/color.mdx:13]:
```suggestion
<img width="960" alt="functional-demo" src="https://user-images.githubusercontent.com/6951037/127494554-07b67dbd-66d3-4d36-8350-5ec837d6d1a5.png" />
```

**Pull request merged**